opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -142,7 +142,8 @@ message Metric {
     IntSum int_sum = 6;
     DoubleSum double_sum = 7;
     IntHistogram int_histogram = 8;
-    DoubleHistogram double_histogram = 9;
+    DeprecatedHistogram deprecated_histogram = 9;
```

Comment:
Nit: Can you add details for how consumers of this should behave (similar to what Bogdan has in his PR)

Reply:
Sure. I'll let that PR merge first, then update the comments. The comment written in that PR more-or-less applies.
I think I should revert "DeprecatedHistogram" back to "DoubleHistogram", leave it the same way IntHistogram keeps its name.